protocol/src/lib.rs

```diff
@@ -26,10 +42,16 @@ pub enum OutgoingMessage {
     Registered(RegisteredMessage),
     /// Notifies listeners that a producer was registered
     #[serde(rename_all = "camelCase")]
-    ProducerAdded { peer_id: String },
+    ProducerAdded {
+        peer_id: String,
+        display_name: Option<String>,
+    },
     /// Notifies listeners that a producer was removed
     #[serde(rename_all = "camelCase")]
-    ProducerRemoved { peer_id: String },
+    ProducerRemoved {
+        peer_id: String,
+        display_name: Option<String>,
```

Comment:
Below you use `#[serde(default)]` on all these but not here. Would be good to keep that consistent

Reply:
So? :)

Reply:
So this is an outgoing message :D